src/components/v5/shared/RichText/RichText.tsx

```diff
@@ -16,28 +21,17 @@ const RichText: FC<RichTextProps> = ({
   toggleOnDecriptionSelect,
   toggleOffDecriptionSelect,
 }) => {
-  const { editorContent, notFormattedContent, field } = useRichText(
-    name,
-    isDecriptionFieldExpanded,
-  );
+  const { editorContent, notFormattedContent, field, characterCount } =
+    useRichText(name, isDecriptionFieldExpanded);
   const { formatMessage } = useIntl();
 
-  const characterCount = editorContent?.storage.characterCount.characters();
-
   return (
     <>
       {editorContent && isDecriptionFieldExpanded ? (
         <>
           <MenuBar editor={editorContent} />
           <EditorContent editor={editorContent} {...field} />
 
-          {characterCount >= MIN_ANNOTATION_NUM &&
-            !isDecriptionFieldExpanded && (
-              <TextButton mode="underlined" onClick={toggleOnDecriptionSelect}>
-                {formatMessage({ id: 'button.expand' })}
-              </TextButton>
-            )}
-
           <div className="flex items-center justify-between mt-4">
```

Comment:
You shouldn't render this div if nothing is rendered inside

Reply:
fixed